docs/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _readme:
+
+syslog-ng-formula
+=================
+
+|img_travis| |img_sr|
+
+.. |img_travis| image:: https://travis-ci.com/saltstack-formulas/syslog-ng-formula.svg?branch=master
+   :alt: Travis CI Build Status
+   :scale: 100%
+   :target: https://travis-ci.com/saltstack-formulas/syslog-ng-formula
+.. |img_sr| image:: https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg
+   :alt: Semantic Release
+   :scale: 100%
+   :target: https://github.com/semantic-release/semantic-release
+
+Formula to set up and configure syslog_ng
+
+.. contents:: **Table of Contents**
+
+General notes
+-------------
+
+See the full `SaltStack Formulas installation and usage instructions
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html>`_.
+
+If you are interested in writing or contributing to formulas, please pay attention to the `Writing Formula Section
+<https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#writing-formulas>`_.
+
+If you want to use this formula, please pay attention to the ``FORMULA`` file and/or ``git tag``,
+which contains the currently released version. This formula is versioned according to `Semantic Versioning <http://semver.org/>`_.
+
+See `Formula Versioning Section <https://docs.saltstack.com/en/latest/topics/development/conventions/formulas.html#versioning>`_ for more details.
+
+Contributing to this repo
+-------------------------
+
+**Commit message formatting is significant!!**
+
+Please see :ref:`How to contribute <CONTRIBUTING>` for more details.
+
+Available states
+----------------
+
+.. contents::
+    :local:
+
+``syslog_ng``
+^^^^^^^^^^^^^
+Installs and configures the syslog_ng package.
+
+``syslog_ng.packages``
+^^^^^^^^^^^^^^^^^^^^^^
+Installs the syslog_ng package and optional packages which may provide additional functionalities.
+
+``syslog_ng.config``
+^^^^^^^^^^^^^^^^^^^^
+This state manages the file ``syslog_ng.conf`` under ``/etc/syslog-ng`` (template found in "syslog_ng/files"). The configuration is populated by values in "syslog_ng/map.jinja" based on the package's default values (and RedHat, Debian, Suse and Arch family distribution specific values), which can then be overridden by values of the same name in pillar.
+
+Testing
+-------
+
+Linux testing is done with ``kitchen-salt``.
+
```

Comment:
Can we get the `Requirements` sub-section here, as merged in https://github.com/saltstack-formulas/template-formula/pull/123?  Of course, if you have anything you want to improve here, please do so.